BLSQ/iaso
# IA-3426 show or unit, project and  user role names in user history log

The UI was showing id, which is not so human friendly

Related JIRA tickets : IA-3426

## Self proofreading checklist


## Doc
NA

## Changes

- Add serializer for `retrieve` method of `/userlogs` endpoint (`ProfileLogsViewset`)
- Make `remove_user_role_name_prefix` method static (`UserRole` model)
- Update API call in front end
- Update `UserLogValue` to extract name from API response
- Add some styling to get links text to display in white if value has been changed (and background is then blue)

## How to test

In iaso> user> management: create or update a user. Change project, location, user role or all 3

Go to users > history and check the latest entry

## Print screen / video

![Screenshot 2024-09-13 at 17 08 05](https://github.com/user-attachments/assets/861d5133-0e32-48e7-8a58-2a2c82117707)
